dandiapi/api/models/asset.py

```diff
@@ -211,25 +211,61 @@ def __str__(self) -> str:
         return self.path
 
     @classmethod
-    def get_path(cls, path_prefix: str, qs: List[str]) -> Set:
+    def get_path(cls, dandiset_pk: str, version: str, path_prefix: str, qs: List[Asset]) -> Dict:
         """
         Return the unique files/directories that directly reside under the specified path.
 
         The specified path must be a folder (must end with a slash).
         """
-        if not path_prefix:
-            path_prefix = '/'
-        prefix_parts = [part for part in path_prefix.split('/') if part]
-        paths = set()
-        for asset in qs:
-            path_parts = [part for part in asset['path'].split('/') if part]
+        # Declare base api url and kwargs for all asset reverse lookups
+        base_api_url = 'https://api.dandiarchive.org'
```

Comment:
how would that work for staging??

Reply:
Ah good catch - it wouldn't work for staging. Maybe the base api url can be set in the `HerokuProductionConfiguration` and `HerokuStagingConfiguration` here https://github.com/dandi/dandi-api/blob/292-paths-endpoint/dandiapi/settings.py#L94

Reply:
That would be great. Citing https://github.com/dandi/dandi-api/issues/286 from above Resolved discussion . I think it should be reasonably simple change to stop postponing fixing it

Reply:
@mvandenburgh, it would be good to check with Brian to get help in evaluating a solution for this problem.

Reply:
@yarikoptic this point is moot with the changes I've recently pushed. We don't return any URLs in the current implementation, and while the point you raised is something we should address, it no longer makes sense to do so in this PR.

Reply:
Great to see hardcoded urls being avoided.